static-site/src/pages/groups.jsx

```diff
@@ -92,31 +119,21 @@ class GroupsPage extends React.Component {
         <GroupCards squashed/>
         <HugeSpacer />
 
-        <ScrollableAnchor id={'datasets'}>
-          <splashStyles.H2>Available Datasets</splashStyles.H2>
-        </ScrollableAnchor>
-        <FlexCenter>
-          <splashStyles.CenteredFocusParagraph>
-            {`Note that this listing is refreshed every ~6 hours.
-            To see a current listing, visit the page for that group by clicking on that group's tile, above.`}
-          </splashStyles.CenteredFocusParagraph>
-        </FlexCenter>
-        <HugeSpacer />
-        {this.state.dataLoaded && (
-          <DatasetSelect
-            datasets={this.state.datasets}
-            columns={datasetColumns({isNarrative: false})}
-          />
-        )}
+        <ListResources
```

Comment:
[haven't reviewed, just noting something while playing with the review app]

Looks like the resource collapsing functionality isn't working (as in, the carot rotates and text changes, but the number of resources shown is unchanged). Or is this some conditional I added if number of cards is 1 or similar?

Reply:
It works for me (Vivaldi using Chrome/124.0.0.0):

https://github.com/nextstrain/nextstrain.org/assets/13424970/832c8db2-5eda-478a-994f-25f130e32410

Reply:
Oh. You've got a bigger screen than me ;) Maybe I set too big of a "show this many when collapsed" count - I remember I made it a function of the number of cards in play. Not a bug in this PR then.

Reply:
> You've got a bigger screen than me

Nah, I just zoomed out a bunch haha